docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,72 @@
+[[proc_updating-booster-deployment-configuration]]
+= Updating the Deployment Configuration of a Booster
+
+The deployment configuration for a booster contains information related to deploying and running the booster in OpenShift, such as route information or readiness probe location. 
+The deployment configuration of a booster is stored in a set YAML files. 
+For boosters that use the Fabric8 Maven Plugin, the YAML files are located in the `src/main/fabric8/` folder. For {NodeJS} boosters using Nodeshift, the YAML files are located in the `.nodeshift` folder.
```

Comment:
Use _directory_ instead of _folder._ (That's Windows terminology.)

Reply:
I think Mac uses _folder_, too.